### ISSUE TITLE
Document mapping between CKEditor 4 and CKEditor 5 plugins

### ISSUE THREAD

**Issue by AnnaTomanek** [ckeditor/ckeditor5#11128]
To make migration from CKEditor 4 easier, we have created the [Migration from CKEditor 4](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/migration/migration-from-ckeditor-4.html) guide - however, it's far from complete. While we do have a table that shows the mapping between CKE4 configuration values and how to achieve something similar with CKE5, we are missing a much more needed mapping between CKE4 plugins and CKE5 plugins/features. Any integrator attempting an upgrade will likely first want to know what is the equivalent of a CKE4 plugin that they used in their setup (or whether there is any available at all).

A tabular form similar to the one with configuration option mappings will probably be a good start.

Let's make sure we link to the most relevant references like feature documentation. Links to npm would be nice to have but they are also included in most feature docs anyway so should not replace these.

**Comment by wimleers** [comment]
YES PLEASE! 🤩 😄

**Comment by wimleers** [comment]
Any news here?

**Comment by AnnaTomanek** [comment]
We should start working on it in around 2 weeks. We will soon be merging a huge restructuring of our Installation documentation section, and all migration docs will land in a new, dedicated space. Once this is done, CKE4->CKE5 migration guides are the next item in our backlog.

**Comment by AnnaTomanek** [comment]
We should start working on it in around 2 weeks. We will soon be merging a huge restructuring of our Installation documentation section, and all migration docs will land in a new, dedicated space. Once this is done, CKE4->CKE5 migration guides are the next item in our backlog.

**Comment by wimleers** [comment]
- Sibling Drupal issue: https://www.drupal.org/project/drupal/issues/3207845
- Drupal.org page listing CKEditor (4) modules: https://www.drupal.org/docs/8/core/modules/ckeditor/contributed-modules-and-plug-ins — broken down by `Drupal modules providing official CKEditor plugins` vs `Drupal modules providing un-official CKEditor plugins` vs `Drupal modules providing Drupal-specific CKEditor plugins`

**Comment by Mgsy** [comment]
Closed by #11605.
